packages/yourloops/app/error.tsx

```diff
@@ -48,17 +52,36 @@ interface OnErrorProps {
   error?: Error
 }
 
+const classes = makeStyles(() => ({
+  errorId: {
+    display: 'block',
+    fontWeight: 'bold'
+  }
+}))
+
 function OnError(props: OnErrorProps): JSX.Element {
   const { t } = useTranslation('yourloops')
   const theme = useTheme()
+  const location = useLocation()
   const [showMore, setShowMore] = React.useState(false)
   const fullScreen = useMediaQuery(theme.breakpoints.down('md'))
-
+  const errorId = uuidv4()
+  const style = classes()
   const errorMessage = props.error?.message ?? 'n/a'
+  const error = props.error ? `Error: ${errorMessage}\nStack: ${props.error.stack}` : 'N/A'
```

Comment:
Why is the fallback error an uppercase `'N/A'` when the fallback error message is lowercase `'n/a'`? Maybe the same value in both cases would be better?